File: people-registry/src/main/java/it/paolomolteni/model/Person.java
package it.paolomolteni.model;

import java.util.List;

/**
 * @author paolo
 *
 */
public class Person {
	
	/**
	 * 
	 */
	public long id;
	
	/**
	 * 
	 */
	public String name;
	
	/**
	 * 
	 */
	public String lastName;
	
	/**
	 * 
	 */
	public Integer age;
	
	/**
	 * 
	 */
	public List<Car> cars;

	/**
	 * 
	 */
	public Person() {
		super();
	}

	/**
	 * @param id
	 * @param name
	 * @param lastName
	 * @param age
	 */
	public Person(long id, String name, String lastName, Integer age) {
		super();
		this.id = id;
		this.name = name;
		this.lastName = lastName;
		this.age = age;
	}
	

}

File: person-registry/src/app/person.ts
import { Car } from './car';

export class Person {
	
	id: number;
  
	name: string;
  
	lastName: string;
  
	age: number;
	
	cars: Car[];
  
	constructor(){}
  
	/*constructor(id : number, name: string, lastName: string, age: number) {
		this.id = id;
        this.name = name;
		this.lastName = lastName;
		this.age = age;
	}*/
	  
}